builtin/logical/pki/path_intermediate.go

```diff
@@ -119,7 +119,7 @@ func (b *backend) pathGenerateIntermediate(ctx context.Context, req *logical.Req
 		// If the operator hasn't configured any of the URLs prior to
 		// generating this issuer, we should add a warning to the response,
 		// informing them they might want to do so and re-generate the issuer.
-		resp.AddWarning("This mount hasn't configured any authority access information fields; this may make it harder for systems to find missing certificates in the chain or to validate revocation status of certificates. Consider updating /config/urls with this information.")
+		resp.AddWarning("This mount hasn't configured any authority access information (AIA) fields; this may make it harder for systems to find missing certificates in the chain or to validate revocation status of certificates. Consider updating /config/urls or the newly generated issuer with this information. Since this certificate is an intermediate, it might be useful to regenerate this certificate after fixing this problem for the root mount.")
```

Comment:
`(AIA)` is being added to this warning, but not several other similarly worded warnings for other endpoints.

However, it's also not really the right term, as it there is only 1 AIA field, and it is actually referring to multiple kinds of URL field besides AIA.

Reply:
I've added `(AIA)` to the other warnings, since it seems we're now in agreement it at least applies to CAs and OCSP servers, and roughly in agreement that it may or may not apply to CRLs too ;-)